adafruit_74hc595.py

```diff
@@ -113,7 +124,12 @@ class ShiftRegister74HC595:
     and indicate the number of shift registers being used
     """
 
-    def __init__(self, spi, latch, number_of_shift_registers=1):
+    def __init__(
+        self,
+        spi: busio.I2C,
```

Comment:
the type for `spi` should be `busio.SPI`

Reply:
Oh well that's just silly on my part, good catch